castervoice/apps/gridsudoku.py

```diff
@@ -0,0 +1,103 @@
+from castervoice.lib.imports import *
```

Comment:
This can be re-factored into `castervoice\rules\apps\mouse_grids`

You will have to define the imports `castervoice.lib.imports` has been removed.